roles/rhcos-post-install/templates/infra-node-machineset.yml.j2

```diff
@@ -36,13 +36,18 @@ items:
               filters: null
               id: {{rhcos_ami_id.stdout}}
             apiVersion: awsproviderconfig.k8s.io/v1alpha1
+            blockDevices:
+            - ebs:
+                iops: 0
```

Comment:
Think we should parameterize the iops field as well.

Reply:
Got it, I'll add them as well